cli.ts

```diff
@@ -430,59 +426,61 @@ const initProject = (
 		map(_ => i18n.__('bootstrapMsg')),
 	);
 
+const runScaffoldPostInit = (scaffoldDir: SanitizedAbsPath.t): Future<TaqError.t, SanitizedAbsPath.t> => {
+	const scaffoldConfigAbspath = SanitizedAbsPath.create(`${scaffoldDir}/scaffold.json`);
+	return pipe(
+		readJsonFile<ScaffoldConfig.t>(scaffoldConfigAbspath),
+		map(logInput('Scaffold Config')),
+		coalesce(_ => ({}) as ScaffoldConfig.t)(identity),
+		chain(scaffoldConfig =>
+			typeof scaffoldConfig === 'object' && scaffoldConfig.postInit
```

Comment:
We don't need to check the type right? It's always an object no?

Reply:
It should be an object, as its man-made and too, external, it cannot be guaranteed. If we ever get third-party scaffolds (which I'm hoping for eventually), then that especially becomes true.